server/ServerNetworking.cpp

```diff
@@ -188,7 +188,13 @@ PlayerConnection::~PlayerConnection() {
                                  << " for player id " << m_ID;
         }
     }
-    m_socket.close();
+
+    // ToDo: Use boost::thread_pool after moving to 1.66.0
```

Comment:
unclear how a thread pool would make sense in this context... you've just got one task that should run detached.

Reply:
Maybe, it doesn't have disconnections happened too often.